cmd/earth/main.go

```diff
@@ -14,11 +15,13 @@ import (
 	"github.com/earthly/earthly/builder"
 	"github.com/earthly/earthly/buildkitd"
 	"github.com/earthly/earthly/cleanup"
+	"github.com/earthly/earthly/config"
 	"github.com/earthly/earthly/conslogging"
 	"github.com/earthly/earthly/domain"
 	"github.com/earthly/earthly/earthfile2llb"
 	"github.com/earthly/earthly/earthfile2llb/variables"
 	"github.com/earthly/earthly/logging"
+
```

Comment:
Nit: Unnecessary blank line?

Reply:
I intentionally added this line as a way to separate earthly imports from thirdparty imports, but we can revert back to having them all together. I have no strong feeling either way.